frontend/app/components/redesign/components/InputField.tsx

```diff
@@ -43,8 +59,17 @@ export const InputField = forwardRef<HTMLInputElement, InputFieldProps>(
           {...props}
         />
         {error && <p className="text-xs text-text-error">{displayError}</p>}
-        {helpText && !error && (
-          <p className="text-xs text-text-secondary">{helpText}</p>
+        {(helpText || showCounter) && !error && (
+          <div className="flex justify-between gap-xs">
+            {helpText && (
+              <p className="text-xs text-text-secondary">{helpText}</p>
+            )}
+            {showCounter && maxLength && (
+              <span className="text-xs leading-xs text-text-secondary">
```

Comment:
There's a minor alignment issue due to line height difference. Let's either remove in both or add in both.

```suggestion
              <span className="text-xs text-text-secondary">
```